packages/platform-server/src/tokens.ts

```diff
@@ -49,6 +49,14 @@ export interface PlatformConfig {
  */
 export const INITIAL_CONFIG = new InjectionToken<PlatformConfig>('Server.INITIAL_CONFIG');
 
+/**
+ * The DI token for setting the server context. This is used to indicate how the page was
+ * generated for analytics purposes.
+ *
+ * @publicApi
+ */
+export const SERVER_CONTEXT = new InjectionToken<string>('Server.SERVER_CONTEXT');
```

Comment:
nit: we can put the default value into the token factory (vs having it at the injection location):
```suggestion
export const SERVER_CONTEXT = new InjectionToken<string>('SERVER_CONTEXT', {
  providedIn: 'root',
  factory: () => 'Angular platform-server'
});
```

Reply:
I considered this at first, but it didn't seem like it was worth the extra KBs tbh. Happy to move it there if you think it's better practice, but it seemed like a lateral move at best.

Reply:
Since this code remains on the server, extra bytes should be ok. My idea was to co-locate the definition and the default value, so that it's all in one place and is easier to manage later. Another approach might be to have an additional constant like `const SERVER_CONTEXT_DEFAULT = 'Angular platform-server';`, so we can have it next to the `SERVER_CONTEXT` one and use in the code where injection happens, but the `factory` approach looks more organic.